### DISCUSSION TITLE
Stability Combinations in SAF

### DISCUSSION THREAD

**Discussion by sonkaste17** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#147] [Ideas]
Need to be investigated

**Comment by MartinSalak** [comment]
Combination for stability calculation should only contain list of load cases and their load factor, no other information is needed. Even though our object `StructuralLoadCombination` is more complex I suggest we only extend its Type enum with `Stability`.

Proposed change to **StructuralLoadCombination**

Changed row:
| Property | Example | Definition |
| --- | --- | --- |
| Type | Envelope<br>Linear<br>Nonlinear<br>**Stability** | For type ‘linear’, the exact content of the combination is presented.<br>For type ‘envelope’, linear combinations can be composed based on relations defined in [StrucutralLoadGroup](https://www.saf.guide/en/latest/loads/structuralloadgroup.html).<br>For type ‘nonlinear’, the exact content of combination is presented. Used for non-linear calculations. Other inputs like initial stress or deformation are to be set in analysis software.<br>**For type 'Stability', the exact content of combination is presented.** |

A simple stability combination might look like this:
**Name:**    `SCOM1`
**Description:**
**Category:**    `Not defined`
**National Standard:**
**Type:**    `Stability`
**Load factor 1:**    `1.35`
**Multiplier 1:**    `1`
**Load case name 1:**    `LC1`
...

It doesn't make sense to allow any other Category than Not defined, but I think it is not necessary to enforce it in SAF documentation

What do you think SAF community?
